pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -306,454 +335,433 @@ var _ = Describe("vmexport", func() {
-		})
+			It("a VirtualMachineExport", func() {
+				err := runDownloadCmd(
+					setFlag(vmexport.PVC_FLAG, pvcName),
+					setFlag(vmexport.VOLUME_FLAG, volumeName),
+					setFlag(vmexport.OUTPUT_FLAG, filepath.Join(GinkgoT().TempDir(), pvcName)),
+					vmexport.INSECURE_FLAG,
+				)
+				Expect(err).ToNot(HaveOccurred())
+			})
 
-		It("Succesfully create and download a VirtualMachineExport with raw format", func() {
-			vmexport.HandleHTTPRequest = func(_ kubecli.KubevirtClient, _ *exportv1.VirtualMachineExport, _ string, _ bool, _ string, _ map[string]string) (*http.Response, error) {
-				resp := http.Response{
-					StatusCode: http.StatusOK,
-					Body: io.NopCloser(bytes.NewReader([]byte{
-						0x1f, 0x8b, 0x08, 0x08, 0xc8, 0x58, 0x13, 0x4a,
-						0x00, 0x03, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x2e,
-						0x74, 0x78, 0x74, 0x00, 0xcb, 0x48, 0xcd, 0xc9,
-						0xc9, 0x57, 0x28, 0xcf, 0x2f, 0xca, 0x49, 0xe1,
-						0x02, 0x00, 0x2d, 0x3b, 0x08, 0xaf, 0x0c, 0x00,
-						0x00, 0x00,
-						0x1f, 0x8b, 0x08, 0x08, 0xc8, 0x58, 0x13, 0x4a,
-						0x00, 0x03, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x2e,
-						0x74, 0x78, 0x74, 0x00, 0xcb, 0x48, 0xcd, 0xc9,
-						0xc9, 0x57, 0x28, 0xcf, 0x2f, 0xca, 0x49, 0xe1,
-						0x02, 0x00, 0x2d, 0x3b, 0x08, 0xaf, 0x0c, 0x00,
-						0x00, 0x00,
-					})),
+			It("a VirtualMachineExport with raw format", func() {
+				vmexport.HandleHTTPRequest = func(_ kubecli.KubevirtClient, _ *exportv1.VirtualMachineExport, _ string, _ bool, _ string, _ map[string]string) (*http.Response, error) {
+					return &http.Response{
+						StatusCode: http.StatusOK,
+						Body: io.NopCloser(bytes.NewReader([]byte{
+							0x1f, 0x8b, 0x08, 0x08, 0xc8, 0x58, 0x13, 0x4a,
+							0x00, 0x03, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x2e,
+							0x74, 0x78, 0x74, 0x00, 0xcb, 0x48, 0xcd, 0xc9,
+							0xc9, 0x57, 0x28, 0xcf, 0x2f, 0xca, 0x49, 0xe1,
+							0x02, 0x00, 0x2d, 0x3b, 0x08, 0xaf, 0x0c, 0x00,
+							0x00, 0x00,
+							0x1f, 0x8b, 0x08, 0x08, 0xc8, 0x58, 0x13, 0x4a,
+							0x00, 0x03, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x2e,
+							0x74, 0x78, 0x74, 0x00, 0xcb, 0x48, 0xcd, 0xc9,
+							0xc9, 0x57, 0x28, 0xcf, 0x2f, 0xca, 0x49, 0xe1,
+							0x02, 0x00, 0x2d, 0x3b, 0x08, 0xaf, 0x0c, 0x00,
+							0x00, 0x00,
+						})),
+					}, nil
 				}
-				return &resp, nil
-			}
 
-			vme := utils.VMExportSpecPVC(vmeName, metav1.NamespaceDefault, "test-pvc", secretName)
-			vme.Status = utils.GetVMEStatus([]exportv1.VirtualMachineExportVolume{
-				{
-					Name:    volumeName,
-					Formats: utils.GetExportVolumeFormat(server.URL, exportv1.KubeVirtGz),
-				},
-			}, secretName)
-			utils.HandleSecretGet(kubeClient, secretName)
-			utils.HandleVMExportCreate(virtClient, vme)
+				err := runDownloadCmd(
+					setFlag(vmexport.FORMAT_FLAG, vmexport.RAW_FORMAT),
+					setFlag(vmexport.PVC_FLAG, pvcName),
+					setFlag(vmexport.VOLUME_FLAG, volumeName),
+					setFlag(vmexport.OUTPUT_FLAG, filepath.Join(GinkgoT().TempDir(), pvcName)),
```

Comment:
nit: Extract `filepath.Join(GinkgoT().TempDir(), pvcName)` to a variable?

Reply:
Same